day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,13 @@
+function speckledFrogs () {
+  for (var i = 10; i > 0; i--) {
```

Comment:
When 1 frog is left in the pool, this prints out `1 speckled frogs sat on a log eating some most delicious bugs.`. How can you modify so that `frog` is singular for this case?